### DISCUSSION TITLE
Lodestar Standup 2024-3 (January 23, 2024)

### DISCUSSION THREAD

**Discussion by philknows** [ChainSafe/lodestar#6313] [Standups]
### Meeting Info

[Jan 23, 2024, 14:00-15:00 UTC](https://savvytime.com/converter/utc-to-germany-berlin-united-kingdom-london-ny-new-york-city-ca-san-francisco-china-shanghai-japan-tokyo-australia-sydney/jan-23-2024/2pm)
Zoom: shared on #lodestar-general ChainSafe Discord channel at 14:00 UTC

### Agenda
- Planning Items
  - v1.15.0 Inclusions
- Team Updates
- Open Discussion/Closing Remarks

### Instructions
Please post here any topics to be added for Standup discussions. Examples of agenda items are issues, discussions, proposals which require feedback, consensus from the team and/or community members.

**Comment by philknows** [comment]
Would like to discuss https://github.com/ChainSafe/lodestar/issues/6137 as we keep getting questions about this and many occurrances of OOM when people are running Holesky.

**Reply by nflaig** [reply]
We should definitely discuss increasing the default memory limit, it looks like we are pretty close to the limit on mainnet. There was also just a memory issue on dappnode https://github.com/dappnode/DAppNodePackage-Lodestar/issues/96. Still need to confirm if this is related to the Nethermind issue from yesterday but I think as this point 4096 is just barely enough on mainnet and any issue will cause the node to crash.

**Reply by matthewkeil** [reply]
Agreed. I think we are running 8gb for holesky and guessing the deneb changes likely increased memory requirements by less than that, but still increased them from capella

**Reply by nflaig** [reply]
We should confirm that `6144` is sufficient for Holesky as well which it is from my observations. We previously just required 8gb because the network was non finalizing. This is still a problem, but as @tuyennhv noted, will be addressed by https://github.com/ChainSafe/lodestar/issues/5968 which leads me to think that 6gb is a safe limit to use on all networks.

**Comment by ensi321** [comment]
@philknows Do you want one final round of confirmation on Electra EIPs from the team before you publish the post https://hackmd.io/aYVbBnQ5SW6QOkifxeANXQ ?

**Comment by twoeths** [comment]
would like to include this in the next release as it caused missed attestation at slot 0 of epoch https://github.com/ChainSafe/lodestar/issues/6268

**Reply by philknows** [reply]
We might have to defer this to v1.16 because we have a deadline today to release a candidate for Sepolia/Holesky forks.

**Comment by philknows** [comment]
Would suggest deferring this to v1.15.1 or v1.16 based on Gajinder's review or even after Deneb? I'm a bit hesitant with changes to block production as we get closer to mainnet. https://github.com/ChainSafe/lodestar/pull/6241

**Comment by philknows** [comment]
Closed in favour of #6349. Notes can be found here: https://github.com/ChainSafe/lodestar/wiki/Lodestar-Planning-&-Standup-Meetings#january-23-2024-planning-and-standup-meeting
